src/main/java/com/minecolonies/coremod/colony/buildings/modules/AbstractCraftingBuildingModule.java

```diff
@@ -148,7 +894,28 @@ public boolean isRecipeCompatible(@NotNull final IGenericRecipe recipe)
         @Override
         public boolean canLearnLargeRecipes() { return false; }
 
+        //todo need to implement this in the respective workers

```

Comment:
It was not intended that this would ever need to return true on the custom module, because this should only be called when trying to learn recipes, and this module can't be taught, it can only use crafterrecipes.